core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -779,7 +788,10 @@ private void handleLocaleCheck() {
     safeDispose();
     super.onDestroy();
     tabCallback = null;
-    hideBackToTopTimer.cancel();
+    if (hideBackToTopTimer != null) {
+      hideBackToTopTimer.cancel();
+    }
+    webViewList.clear();
```

Comment:
is this necessary?

Reply:
not anymore I think. I added the null-check because the hideBackToTopTimer was not initalized in onCreate but as a property.